index.html

```diff
@@ -105,7 +105,7 @@ <h4>Math Ignite</h4>
     <section id="speakers">
         <div id="featured-speakers" class="speaker-card"></div>
```

Comment:
Great design @Eccb7 🤝

- [ ] [Optional] I would recommend you implement the `more` button as given in the design guidelines.
![More button](https://github.com/Eccb7/Capstone_Project-1/assets/63917725/dc32db3a-8611-4cc5-be78-b15d2f516034)